cmd/storj/cmd/rm.go

```diff
@@ -32,26 +32,37 @@ func init() {
 func delete(cmd *cobra.Command, args []string) error {
 	ctx := process.Ctx(cmd)
 
+	identity, err := rmCfg.Load()
+	if err != nil {
+		return err
+	}
+
+	bs, err := rmCfg.GetBucketStore(ctx, identity)
+	if err != nil {
+		return err
+	}
+
 	if len(args) == 0 {
-		return errs.New("No object specified for deletion")
+		fmt.Println("No object specified for deletion")
+		return nil
 	}
 
-	so, err := getStorjObjects(ctx, rmCfg)
+	u, err := utils.ParseURL(args[0])
 	if err != nil {
 		return err
 	}
 
-	u, err := url.Parse(args[0])
+	o, err := bs.GetObjectStore(ctx, u.Host)
```

Comment:
We should check if both `u.Host` and `u.Path` are not empty before using them. Otherwise we can delete a bucket with this command.

Reply:
Done